packages/hawtio/src/plugins/camel/trace/Trace.tsx

```diff
@@ -199,84 +189,85 @@ export const Trace: React.FunctionComponent = () => {
           {!isTracing ? 'Start Tracing' : 'Stop Tracing'}
         </Button>
       </PanelHeader>
-      <CardBody>
-        {!isTracing && (
-          <React.Fragment>
-            <Text className='noTracing' data-testid='no-tracing' component='p'>
-              Tracing allows you to send messages to a route and then step through and see the messages flow through a
-              route to aid debugging and to help diagnose issues.
-            </Text>
-            <Text className='noTracing' data-testid='no-tracing' component='p'>
-              Once you start tracing, you can send messages to the input endpoints, then come back to this page and see
-              the flow of messages through your route.
-            </Text>
-            <Text className='noTracing' data-testid='no-tracing' component='p'>
-              As you click on the message table, you can see which node in the flow it came through; moving the
-              selection up and down in the message table lets you see the flow of the message through the diagram.
-            </Text>
-          </React.Fragment>
-        )}
-        {isTracing && (
-          <React.Fragment>
-            <Panel id='route-diagram-tracing-view' isScrollable>
-              <PanelMain>
-                <PanelMainBody>
-                  <RouteDiagramContext.Provider
-                    value={{
-                      graphNodeData,
-                      setGraphNodeData,
-                      graphSelection,
-                      setGraphSelection,
-                      setShowStatistics,
-                    }}
-                  >
-                    <RouteDiagram />
-                  </RouteDiagramContext.Provider>
-                </PanelMainBody>
-              </PanelMain>
-            </Panel>
-
+      <PanelMain>
+        <PanelMainBody>
+          {!isTracing && (
+            <React.Fragment>
+              <Text className='noTracing' data-testid='no-tracing' component='p'>
```

Comment:
Are not the data-testid attributes supposed to have unique values? The 3 <Text> components here have the same id.